src/modules/aws_account@0.0.1/entity/aws_account.ts

```diff
@@ -4,11 +4,14 @@ import {
   PrimaryGeneratedColumn,
 } from 'typeorm'
 
+import { cloudId, } from '../../../services/cloud-id'
+
 @Entity({
   name: 'aws_account',
 })
 export class AwsAccountEntity {
   @PrimaryGeneratedColumn()
+  @cloudId
```

Comment:
I guess this never goes in the cloud, but we have to pick one?

Reply:
Yeah. If I want to not provide a custom `entityId` I have to pick a column. That's all there was to this, basically.